.github/PULL_REQUEST_TEMPLATE/release_template.md

```diff
@@ -1,14 +1,12 @@
-Link to the milestone on Github e.g. https://github.com/nimblehq/git-templates/milestone/41?closed=1
-or
-Link to the project management tool for the release
+Link to the milestone on Github e.g. https://github.com/nimblehq/flutter-templates/milestone/41?closed=1
+or Link to the project management tool for the release
 
 ## Features
 
 Provide the ID and title of the issue in the section for each type (feature, chore and bug). The link is optional.
 
-- [sc-1234] As a user, I can log in
-or
-- [[sc-1234](https://app.shortcut.com/nimblehq/story/1234)] As a user, I can log in
+- [#1] As a user, I can log in or
+- [[#1](https://github.com/nimblehq/flutter-templates/issues/1)] As a user, I can log in
```

Comment:
```suggestion
- [[#1](https://github.com/nimblehq/ic-flutter-avishek/issues/1)] As a user, I can log in
```

Reply:
Updated in https://github.com/nimblehq/ic-flutter-avishek/pull/26/commits/087f4c2fa27bd775ebe3c3f2bbaaf109f6dbd957.